hack/allocate.sh

```diff
@@ -73,12 +73,12 @@ EOF
 serving() {
   echo "${em}② Knative Serving${me}"
 
-  kubectl apply --filename https://github.com/knative/serving/releases/download/$knative_version/serving-crds.yaml
+  kubectl apply --filename https://github.com/knative/serving/releases/download/knative-$knative_version/serving-crds.yaml
 
   sleep 5
   kubectl wait --for=condition=Established --all crd --timeout=5m
 
-  curl -L -s https://github.com/knative/serving/releases/download/$knative_version/serving-core.yaml | yq 'del(.spec.template.spec.containers[]?.resources)' -y | yq 'del(.metadata.annotations."knative.dev/example-checksum")' -y | kubectl apply -f -
+  curl -L -s https://github.com/knative/serving/releases/download/knative-$knative_version/serving-core.yaml | yq 'del(.spec.template.spec.containers[]?.resources)' -y | yq 'del(.metadata.annotations."knative.dev/example-checksum")' -y | yq | kubectl apply -f -
```

Comment:
Nice to see the checksum removal is no longer necessary
Bummer to see the re-pipe through yq is!
🤷🏻